src/Sulu/Bundle/ContentBundle/Tests/Functional/Command/CleanupHistoryCommandTest.php

```diff
@@ -47,14 +49,34 @@ public function setUp()
         $this->tester = new CommandTester($cleanupCommand);
     }
 
+    private function createContentNode(NodeInterface$parent, $webspaceKey, $title)
+    {
+        $node = $parent->addNode($title);
+        $node->setProperty('i18n:en-template', 'default');
+        $node->setProperty('i18n:en-creator', 1);
+        $node->setProperty('i18n:en-created', new \DateTime());
+        $node->setProperty('i18n:en-changer', 1);
+        $node->setProperty('i18n:en-changed', new \DateTime());
+        $node->setProperty('i18n:en-title', 'Homepage');
+        $node->setProperty('i18n:en-state', WorkflowStage::PUBLISHED);
+        $node->setProperty('i18n:en-published', new \DateTime());
+        $node->setProperty('i18n:en-url', '/');
+        $node->addMixin('sulu:page');
+
+        return $node;
+    }
+
     private function initNoHistory($webspaceKey, $locale)
     {
         $this->initPhpcr();
 
         $contentNode = $this->sessionManager->getContentNode($webspaceKey);
-        $this->phpcrStrategy->save($contentNode, '/team', 1, $webspaceKey, $locale);
-        $this->phpcrStrategy->save($contentNode, '/team/daniel', 1, $webspaceKey, $locale);
-        $this->phpcrStrategy->save($contentNode, '/team/johannes', 1, $webspaceKey, $locale);
+        $teamNode = $this->createContentNode($contentNode, $webspaceKey, 'team');
+        $this->phpcrStrategy->save($teamNode, '/team', 1, $webspaceKey, $locale);
+        $danielNode = $this->createContentNode($teamNode, $webspaceKey, 'daniel');
+        $this->phpcrStrategy->save($danielNode, '/team/daniel', 1, $webspaceKey, $locale);
+        $johannesNode = $this->createContentNode($teamNode, $webspaceKey, 'johannes');
+        $this->phpcrStrategy->save($johannesNode, '/team/johannes', 1, $webspaceKey, $locale);
```

Comment:
This was necessary because the test was a bit faked. The routes have now really to be attached to a node, because the `move` method in the `RlpStrategy` doesn't exist anymore, and if a route is moved or changed is determined by the passed content node.